### PR TITLE
Verify: remove --generator=run-pod/v1 obsolete flag

### LINKED ISSUE #617: --generator=run-pod/v1 obsolete
When following the user guide https://submariner.io/operations/usage/  the --generator=run-pod/v1  is not supported for kubectl anymore, making the command fail. Would be good to update docs.

### PR DESCRIPTION
**Pull request by nyechiel**
Closes https://github.com/submariner-io/submariner-website/issues/617

Signed-off-by: nyechiel <nyechiel@redhat.com>

### CODE REVIEW

File: src/content/operations/usage/_index.en.md [hunk 1]
```diff
@@ -551,7 +551,7 @@ Events:                    <none>
 Run a test Pod on **cluster2** and try to access the `nginx` Service from within the Pod:
 
 ```bash
-$ kubectl run -n nginx-test \tmp-shell --rm -i --tty --image quay.io/submariner/nettest -- /bin/bash
+kubectl run -n nginx-test \tmp-shell --rm -i --tty --image quay.io/submariner/nettest -- /bin/bash
```
Comment (sridhargaddam) [inline src/content/operations/usage/_index.en.md]:
```suggestion
kubectl run -n nginx-test tmp-shell --rm -i --tty --image quay.io/submariner/nettest -- /bin/bash
```

**Review by sridhargaddam — CHANGES_REQUESTED** [review]
We can remove the "\"

**Review by sridhargaddam — APPROVED** [review]
Thanks @nyechiel

**Review by skitt — COMMENTED** [review]
Why are the `--kubeconfig` options being removed?

**Comment by nyechiel** [comment]
> Why are the `--kubeconfig` options being removed?

Good catch, that's a mistake! I will fix it

File: src/content/getting-started/quickstart/kind/_index.md [hunk 2]
```diff
@@ -148,8 +148,8 @@ subctl export service --kubeconfig output/kubeconfigs/kind-config-cluster2 --nam
 Run `nettest` from `cluster1` to access the `nginx` service:
 
 ```bash
-kubectl --kubeconfig output/kubeconfigs/kind-config-cluster1 -n default  run --generator=run-pod/v1 \
-tmp-shell --rm -i --tty --image quay.io/submariner/nettest -- /bin/bash
+kubectl --kubeconfig output/kubeconfigs/kind-config-cluster1 -n default run tmp-shell --rm -i --tty --image quay.io/submariner/nettest
```
Comment (skitt) [inline src/content/getting-started/quickstart/kind/_index.md]: This needs a continuation:

```suggestion
kubectl --kubeconfig output/kubeconfigs/kind-config-cluster1 -n default run tmp-shell --rm -i --tty --image quay.io/submariner/nettest \
```

**Review by skitt — CHANGES_REQUESTED** [review]

**Review by skitt — APPROVED** [review]

**Pull request merged**